python/src/main/Consume.py

```diff
@@ -1,13 +1,19 @@
-/* Copyright (c) 2023 & onwards. Hewlett Packard Enterprise Company, All rights reserved */
+#Kafka-python consumer exmaple
 from kafka import KafkaConsumer
 import sys
 import time
 
+#Accepts user provided DataAccessGateway/Broker IP or hostname followed by a port number.
+#The IP address/hostname and port number are separted by a colon "<localhost:9092>".
 server = sys.argv[1]
-print("Broker: " + server)
+print("DataAccessGateway/Broker: " + server)
```

Comment:
It would be much better to use standard argument parsing. Here is an example:
```
import argparse
import re

parser = argparse.ArgumentParser(
                    prog='producer',
                    description='Generates a number of messages and sends them to a particular Kafka topic')
parser.add_argument('-c', '--count', default=100, help='Number of messages to send (default 100)')
parser.add_argument('-b', '--broker', help='host:port for the bootstrap server', required=True)
parser.add_argument('-t', '--topic', default='foo', help='The topic where the messages should go')

args = parser.parse_args()

if not re.match("[^ :]+:\\d+", args.broker):
    print("Hostname should be in the form host:port")
    parser.print_help()
```

Reply:
Sure Ted. Will add.